src/styles/globals.css

```diff
@@ -25,7 +25,7 @@
   h1 {
     @apply text-3xl;
     @apply text-black;
-    @apply font-medium;
+    @apply font-bold;
```

Comment:
be careful of this affecting other pages

Reply:
I double checked figma and all the places where we use the h1 tag and it actually does need to be bolded everywhere :)